toponetx/algorithms/exterior_calculus/exterior_calc.py

```diff
@@ -0,0 +1,349 @@
+"""
+Exterior calculus operators built on TopoNetX coincidence matrices.
+
+This module exposes `ExteriorCalculusOperators`, a user-facing wrapper around:
+- topological coboundaries `d_k`,
+- diagonal Hodge stars `*_k`,
+- codifferentials `delta_k`,
+- DEC Hodge Laplacians `Delta_k`.
+
+For triangle meshes embedded in R^3, additional surface FEM operators on 0-forms
+are available through the chosen metric backend.
+"""
+
+from __future__ import annotations
```

Comment:
I couldn't find a reason why you need to import future annotation support in this module?